src/Task1/SelectGenre.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from "react";
+import "./selectGenre.css";
+const SelectGenre = () => {
+  const [show, setShow] = useState("");
+  const [clip, setClippath] = useState("polygon(50% 75%, 0 25%, 100% 25%)");
+  return (
+    <div style={{ display: "flex", justifyContent: "center" }}>
+      <div
+        id="list1"
+        className={"dropdown-check-list " + show}
+        tabindex="100"
+        style={{ width: "400px", backgroundColor: "#232323", color: "#FFFFFF" }}
+      >
+        <span
```

Comment:
This should probably be a `<button>` tag. And after that, you probably even don't need to set `tabIndex` for the wrapper `<div>`, as this button is actually what should be interact-able.

Reply:
I removed tabindex